README.md

```diff
@@ -285,16 +285,9 @@ There should be a built Debian package (a `.deb` file) in your project directory
 > *My volume on Spotify is 100% and it's still too quiet!*
 
 Have you tried turning the volume up using the command `alsamixer`?
-
-No satisfactory results? Try adding normalisation-pregain to the VOLUME_ARGS
-to higher the initial volume
-
-```
--normalisation-pregain 2
-```
-
-NOTE: dB is a logarithmic scale, so small values have a lot more impact
-than in a linear one.
```

Comment:
This is horrible advice.

Especially considering:

The new default normalisation-method is dynamic in libresot >= 0.2, meaning it actually uses a limiter. A normalisation-pregain of 20 would just smash against that limiter and be a distorted mess. Even without a limiter with the basic mode the gain would just be reduced automatically to not clip. So basically all you would be doing is spamming the logs with warnings about how you should reduce the gain.

The max normalisation-pregain will soon be 10.0 for the above reason. Even 10.0 is a crazy basically unrealistically high number. The usable positive range maxes out at maybe 6.0 IMHO.